packages/edge-config/src/index.ts

```diff
@@ -164,7 +164,12 @@ export function createClient(
 
   const baseUrl = connection.baseUrl;
   const version = connection.version; // version of the edge config read access api we talk to
-  const headers = { Authorization: `Bearer ${connection.token}` };
+  const headers: Record<string, string> = {
+    Authorization: `Bearer ${connection.token}`,
+  };
+
+  if (typeof process !== 'undefined' && process.env.NODE_ENV)
```

Comment:
Should we also add the edge config SDKs version to the request headers here?

Reply:
Added